src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -746,20 +715,18 @@ public static void readFromNBT(@NotNull final NBTTagCompound compound)
         for (int i = 0; i < colonyTags.tagCount(); ++i)
         {
             @NotNull final Colony colony = Colony.loadColony(colonyTags.getCompoundTagAt(i));
-            colonies.put(colony.getID(), colony);
+            colonies.add(colony);
 
-            if (!coloniesByWorld.containsKey(colony.getDimension()))
-            {
-                coloniesByWorld.put(colony.getDimension(), new ArrayList<>());
-            }
-            coloniesByWorld.get(colony.getDimension()).add(colony);
-
-            topColonyId = Math.max(topColonyId, colony.getID());
+            addColonyByWorld(colony);
         }
 
         Log.getLogger().info(String.format("Loaded %d colonies", colonies.size()));
     }
 
+    private static void addColonyByWorld(Colony colony) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)